### PR TITLE
Dashboard chart

### PR DESCRIPTION
**Pull request by prwelber**

### CODE REVIEW

**Comment by prwelber** [comment]
Todo:
- [ ] mobile view
- [ ] ethereum
- [ ] cache 'week'
- [ ] fix dates when 'all' is selected

**Comment by prwelber** [comment]
depends on https://github.com/blockchain/My-Wallet-V3/pull/438

File: app/partials/home.pug [hunk 1]
```diff
@@ -1,34 +1,51 @@
...
+    .border.flex-column.pal.chart-control(ng-controller="PriceChartController")
+      .flex-row.f-24.blue.em-300.pb-15.f-16-mobile
+        a.blue.opacity-5(ng-click="setCurrency('btc')" ng-class="{'opacity-10': isCurrency('btc')}")
+          span BTC =&nbsp;
+          fiat(btc="BTCCurrency.conversion")
+        a.blue.opacity-5(ng-click="setCurrency('eth')" ng-class="{'opacity-10': isCurrency('eth')}")
+          span.ml-15 ETH =&nbsp;
+          fiat(eth="1")
+      .flex-column.flex-center.flex-justify(ng-if="!options")
+        <img src="img/spinner.gif" width="35" />
```
Comment (plondon) [inline app/partials/home.pug]: Angularize?

File: assets/js/controllers/priceChart.controller.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+function PriceChartController ($scope, MyBlockchainApi, Wallet, currency, localStorageService, $timeout) {
+  // scale in seconds
+  const FIFTEENMIN = 15 * 60;
+  const HOUR = 60 * 60;
+  const TWOHOUR = 2 * 60 * 60;
+  const DAY = 24 * 60 * 60;
+  const FIVEDAY = 5 * 24 * 60 * 60;
+  const BTCSTART = 1282089600;
+  const ETHSTART = 1438992000;
```
Comment (plondon) [inline assets/js/controllers/priceChart.controller.js]: Should these be hardcoded?

Comment (plondon) [reply assets/js/controllers/priceChart.controller.js]: I see, this is for all time?

File: app/partials/home.pug [hunk 3]
```diff
@@ -1,34 +1,51 @@
...
-          li.flex-between.border-bottom-light.mid-grey(ng-if="activeLegacyAddresses().length > 0")
-            span.f-16.em-300.flex-center(translate="IMPORTED_ADDRESSES")
-            .flex-column.flex-align-end
-              span.f-16.em-300 {{ getLegacyTotal() | toBitCurrency:BTCCurrency }}
-              fiat(btc="getLegacyTotal()")
-          li.flex-between.border-bottom-light
-            span.f-16.em-300.flex-center {{ eth.defaultAccount.label }}
-            .flex-column.flex-align-end
-              span.f-16.em-300 {{ eth.total() | format:'eth' }} ETH
-              fiat.f-12.em-300(eth="eth.total()")
```
Comment (plondon) [inline app/partials/home.pug]: I think we should still show My Ether Wallet even if balance is 0

Comment (plondon) [reply app/partials/home.pug]: Would also rather use defaultAccount.label than translation

File: app/partials/home.pug (hunk 1 above)
Comment (prwelber) [reply app/partials/home.pug]: Yup. Weird that I did that haha.

File: app/partials/home.pug (hunk 3 above)
Comment (prwelber) [reply app/partials/home.pug]: I think this got messed up in a merge. Will add back.

**Pull request merged**